src/Fantomas/FantomasConfig.fs

```diff
@@ -0,0 +1,146 @@
+﻿module Fantomas.FantomasConfig
+
+open System
+open System.IO
+open Fantomas.FormatConfig
+open Newtonsoft.Json
+
+module Dto =
+
+    /// DTO for FormatConfig (we want to keep this separated from FormatConfig in case we change the FormatConfig internal type)
+    [<CLIMutable>]
+    type FantomasConfig = 
```

Comment:
Do we really need separate type? I think it is OK to have warnings from loading JSON if we remove some config option.